service/geopm-service.spec.in

```diff
@@ -61,6 +64,12 @@ Prefix: %{_prefix}
 %define docdir %{_defaultdocdir}/geopm-%{version}
 %endif
 
+%if 0%{?el7}
+%global python3_pkgversion 36
+%else
+%{!?python3_pkgversion:%global python3_pkgversion 3}
```

Comment:
So if we're on RHEL 7, use Python 3.6 (that's what 36 means here?).  Otherwise, use Python 3 for all other distros?

Wouldn't we want Python >= 3.6 in all cases?

Reply:
This is just the different ways that RHEL 7 and RHEL 8 identify python packages.  For RHEL 7 the minor number is specified, for RHEL 8 it is not.